### PR TITLE
Introduce ringbuf_getnb.

### PR DESCRIPTION
**Pull request by MichalBlk**
This function is required by #935.

### CODE REVIEW

**Review by wiklam — APPROVED** [review]

File: include/sys/ringbuf.h [hunk 1]
```diff
@@ -26,6 +26,7 @@ bool ringbuf_putb(ringbuf_t *buf, uint8_t byte);
 /*! \brief Put exactly n bytes into buf if there's enough space. */
 bool ringbuf_putnb(ringbuf_t *buf, uint8_t *data, size_t n);
 bool ringbuf_getb(ringbuf_t *buf, uint8_t *byte_p);
+bool ringbuf_getnb(ringbuf_t *buf, uint8_t *data, size_t n);
```
Comment (wiklam) [inline include/sys/ringbuf.h:29]: I would provide a comment similar to `ringbuf_putnb`

Comment (MichalBlk) [reply include/sys/ringbuf.h:30]: Done.

**Review by pj1031999 — APPROVED** [review]
LGTM

**Review by cahirwpz — APPROVED** [review]
LGTM

**Pull request merged**